### DISCUSSION TITLE
`tracing-subscriber` how to suppress logging from specific target

### DISCUSSION THREAD

**Discussion by BratSinot** [tokio-rs/tracing#2633] [Q&A]
Greetings!

Is there any way to suppress logging from specific target? `none` and `off` doesn't work for target:
```
RUST_LOG=log=none
RUST_LOG=log=off
```

Of course, I can set `none` for all targets and set level individually, but this not very convient:
```
RUST_LOG=none,hyper=error,my_target=trace,<...>
```

**Comment by davidbarsky** [comment]
I'd suggest using using [`filter_fn`](https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/fn.filter_fn.html) instead of `EnvFilter` directly. You should be able to do what you're looking to do with that.
